glean-core/python/glean/glean.py

```diff
@@ -189,12 +202,17 @@ def _initialize_core_metrics(cls):
         """
         Set a few metrics that will be sent as part of every ping.
         """
+        from . import _builtins
+
         # TODO: 1594184
-        pass
+        _builtins.metrics.glean
```

Comment:
Is this triggering the load of builtin metrics from registry files? If so, any chance you could add a small comment about this?